src/App.jsx

```diff
@@ -1,16 +1,29 @@
 import React from 'react';
+import { Sum } from './components/Sum';
 import './App.scss';
 
 const App = () => (
-  <>
-    <p>Sum of 2 and 3 is 5</p>
-    <p>Sum of -5 and 5 is 0</p>
-    <p>Sum of 10 and 0 is 10</p>
-    <p>Sum of 0 and 5 is 5</p>
-    <p>Sum of 0 and 0 is 0</p>
-    {/* Replace paragraphs with Sum componets */}
-    {/* And remove comment lines :) */}
-  </>
+  <div>
+    <Sum
+      a={2}
+      b={3}
+    />
```

Comment:
не переноси если пропсы короткие и помещаются в одну строку
```suggestion
    <Sum a={2} b={3} />
```